### PR TITLE
Misha19.1var1 endpoint for get active invite list of users from workspace

### PR DESCRIPTION
**Pull request by DoctorHeisenberg**
Happy flow:
1. Craeate new workspace, or open old
2. Make sure that you are owner or manager of him
3. Send invite to one or more users
4. Send the HTTP request to "activeinvites/{workspaceId}/info"
5. Watch the result (list of active invites)

### CODE REVIEW

File: Provis.Core/Services/WorkspaceService.cs [hunk 1]
```diff
@@ -13,6 +13,7 @@
 using System.Threading.Tasks;
 using System.Collections.Generic;
 using Provis.Core.Helpers.Mails;
+using Provis.Core.DTO.userDTO;
```
Comment (YevheniiVovk) [inline Provis.Core/Services/WorkspaceService.cs]: UserDTO naming should start from upper case character

File: Provis.Core/Services/WorkspaceService.cs [hunk 2]
```diff
@@ -265,5 +266,29 @@ public async Task<WorkspaceInfoDTO> GetWorkspaceInfoAsync(int workspId, string u
 
             return workspace;
         }
+
+        public async Task<List<UserInviteInfoDTO>> 
+            GetWorkspaceActiveInvitesAsync(int workspId, string userId)
+        {
+            var workspace = _workspaceRepository
+                .Query()
+                .FirstOrDefault(x => x.Id == workspId);
```
Comment (YevheniiVovk) [inline Provis.Core/Services/WorkspaceService.cs]: Please use GetByKey

File: Provis.Core/Services/WorkspaceService.cs [hunk 3]
```diff
@@ -265,5 +266,29 @@ public async Task<WorkspaceInfoDTO> GetWorkspaceInfoAsync(int workspId, string u
...
+            {
+                throw new HttpException(System.Net.HttpStatusCode.NotFound, 
+                    "Workspace with this ID not exist");
+            }
+
+            var invitesList = await _inviteUserRepository
+                .Query()
+                .Include(x => x.FromUser)
+                .Where(x => x.WorkspaceId == workspId && x.FromUserId == userId && x.IsConfirm == null)
+                .ToListAsync();
```
Comment (YevheniiVovk) [inline Provis.Core/Services/WorkspaceService.cs]: Logical miss we should get all invite from workspace when user role is owner for current

File: Provis.WebApi/Controllers/WorkspaceController.cs [hunk 4]
```diff
@@ -92,5 +92,17 @@ public async Task<IActionResult> GetWorkspaceInfoAsync(int id)
 
             return Ok(workspInfo);
         }
+
+        [HttpGet]
+        [Authorize]
+        [WorkspaceRoles(new WorkSpaceRoles[] { WorkSpaceRoles.OwnerId, WorkSpaceRoles.ManagerId })]
+        [Route("activeinvites/{workspaceId}/info")]
```
Comment (AndriyKovalets) [inline Provis.WebApi/Controllers/WorkspaceController.cs]: please change route (example: "{workspaceId}/invite/active")

File: Provis.Core/Entities/User.cs [hunk 5]
```diff
@@ -10,6 +10,8 @@ public class User : IdentityUser, IBaseEntity
         public string Name { get; set; }
         public string Surname { get; set; }
 
+        public string Img { get; set; }
```
Comment (YevheniiVovk) [inline Provis.Core/Entities/User.cs]: NAMING 
I think that this field should be named as RelativeImgPath or RelativeAvatarPath

File: Provis.Core/Entities/UserTask.cs [hunk 6]
```diff
@@ -14,6 +14,8 @@ public class UserTask : IBaseEntity
 
         public User User { get; set; }
 
+        public bool IsUserDeleted { get; set; }
```
Comment (YevheniiVovk) [inline Provis.Core/Entities/UserTask.cs]: Please create a tech dept to investigate what indexes should be added for each of generated EF queries use seek but not scan, after that make a propose, confirm it and create a new task for adding indexes

**Review by YevheniiVovk — APPROVED** [review]

File: Provis.Core/Entities/User.cs (hunk 5 above)
Comment (AndriyKovalets) [reply Provis.Core/Entities/User.cs]: tech dept: https://sharing.clickup.com/t/h/1tf8nxb/PD6KF7ZWHU8F0ZS

File: Provis.Core/Entities/UserTask.cs (hunk 6 above)
Comment (DoctorHeisenberg) [reply Provis.Core/Entities/UserTask.cs]: tech dept: https://sharing.clickup.com/t/h/1tf8p4b/8TT93ZUY8PVZKHG

**Review by AndriyKovalets — APPROVED** [review]

**Pull request merged**